content/en/blog/2024/humans-of-otel-na-2024.md

```diff
@@ -0,0 +1,522 @@
+---
+title: The Humans OpenTelemetry - KubeCon NA 2024
+linkTitle: Humans of OTel NA 2024
+date: 2024-12-17
+author: >-
+  [Adriana Villela](https://github.com/avillela) (Dynatrace),
+issue: 5812
+sig: End User SIG
+# prettier-ignore
+cSpell:ignore: Adriana Akhigbe Bhattacharya Bluesky brainer Braydon Budha causely Christos commoditize Creeden David Endre Eromosele Gohberg google hazel Kains Lee Luna Markou Miguel nivenly Outreachy Reese Sara sematext Sharr TIMgKXCeiyQ traducción traduction transformative tyk weakly youtube
+---
+
+We're back with our third edition of
+[Humans of OpenTelemetry](/blog/2024/humans-of-otel-eu-2024/), this time from
+KubeCon NA in Salt Lake City, Utah, USA. Once again,
+[Reese Lee](https://github.com/reese-lee) and I interviewed OpenTelemetry
+contributors and end users (and each other!), and learned how they got involved
+with OTel:
+
+- [Hazel Weakly (The Nivenly Foundation)](https://github.com/hazelweakly)
+- [Eromosele Akhigbe (Sematext)](https://github.com/AkhigbeEromo)
+- [Budha Bhattacharya (Tyk)](https://github.com/hellobudha)
+- [Miguel Luna (Elastic)](https://github.com/mlunadia)
+- [Adriana Villela (Dynatrace)](https://github.com/avillela)
+- [David Gohberg (Monday)](https://github.com/dattto)
+- [Endre Sara (Causely)](https://github.com/esara)
+- [Braydon Kains (Google)](https://github.com/braydonk)
+- [Christos Markou (Elastic)](https://github.com/ChrsMark)
+- [Reese Lee (New Relic)](https://github.com/reese-lee)
+
+Also, special thanks to:
+
+- [Reese Lee](https://github.com/reese-lee), my co-interviewer
+- [Henrik Rexed](https://github.com/henrikrexed) for providing the audio and
+  video recording equipment, and doing the initial edits of the raw footage
+
+You can watch the full recording here:
```

Comment:
```suggestion
## Vide

You can watch the full recording here:
```

you might want to add a section title above the video as well so if someone scrolls down in the transcript they can use that link on the right menu to go back to the video